internal/controller/controller.go

```diff
@@ -0,0 +1,133 @@
+package controller
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/Painkiller675/url_shortener_6750/internal/config"
+	"github.com/Painkiller675/url_shortener_6750/internal/repository"
+	"github.com/Painkiller675/url_shortener_6750/internal/service"
+	"go.uber.org/zap"
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+type Controller struct {
+	logger  *zap.Logger
+	storage *repository.Storage
+}
+
+func New(logger *zap.Logger, storage *repository.Storage) *Controller {
+	return &Controller{logger: logger, storage: storage}
+}
+
+func (c *Controller) CreateShortURLHandler(res http.ResponseWriter, req *http.Request) {
+
+	body, err := io.ReadAll(req.Body)
+	if err != nil || len(body) == 0 {
+		http.Error(res, http.StatusText(http.StatusBadRequest), http.StatusBadRequest)
+		return
+	}
+
+	// write an alias
+	randAl := service.GetRandString(8)
+	c.storage.SafeStorage.StoreAlURL(randAl, string(body))
+	// response molding
+	baseURL := config.StartOptions.BaseURL
+	fmt.Println("BASE URL in CreateNoJS = ", baseURL)
```

Comment:
```suggestion
	c.logger.Debug("BASE URL in CreateNoJS = ", baseURL)
```
на этом этапе уже не следует оставлять fmt.

Reply:
done